apisix/plugins/cors.lua

```diff
@@ -76,6 +76,30 @@ local _M = {
     schema = schema,
 }
 
+local function create_mutiple_origin_cache(conf)
+    if not str_find(conf.allow_origins, ",", 1, true) then
+        return nil
+    end
+    local origin_cache = {}
+    local iterator, err = re_gmatch(conf.allow_origins, "([^,]+)", "jiox")
+    if not iterator then
+        core.log.error("match origins failed: " .. err)
+        return nil
+    end
+    while true do
+        local origin, err = iterator()
+        if err then
+            core.log.error("iterate origins failed: " .. err)
```

Comment:
optimization: avoid to concat string object.

`core.log.error("iterate origins failed: ", err)`